srcpkgs/gpgme/template

```diff
@@ -13,17 +13,17 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://www.gnupg.org/software/gpgme/index.html"
 distfiles="https://www.gnupg.org/ftp/gcrypt/gpgme/gpgme-${version}.tar.bz2"
-checksum=361d4eae47ce925dba0ea569af40e7b52c645c4ae2e65e5621bf1b6cdd8b0e9e
+checksum=25a5785a5da356689001440926b94e967d02e13c49eb7743e35ef0cf22e42750
+
+CXXFLAGS=" -D_GLIBCXX_USE_C99_STDIO=1"
 
 if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
 	configure_args+=" ac_cv_sys_file_offset_bits=no"
 elif [ "$XBPS_TARGET_WORDSIZE" = "32" ]; then
 	CFLAGS="-D_FILE_OFFSET_BITS=64 -DLARGEFILE_SOURCE=1"
-	CXXFLAGS="${CFLAGS}"
+	CXXFLAGS+="${CFLAGS}"
```

Comment:
there needs to be a space if you append.

Reply:
Good catch.